storage/storage.go

```diff
@@ -1981,6 +1981,12 @@ func (c composeSourceObj) IfGenerationMatch(gen int64) {
 	}
 }
 
+const (
+	// The AES256 encryption algorithm used with the Customer-Supplied
+	// Encryption Keys feature.
+	aes256Algorithm = "AES256"
```

Comment:
I was looking in `storage.go` and didn't see any constants at the top of the file. It seems like constants are declared nearby where they are used. Not sure if I'm reading this correctly.

Reply:
It is defined as a block at the top of the file: https://github.com/googleapis/google-cloud-go/blob/main/storage/storage.go#L75-L87

Reply:
Ah okay, thanks! I've added the 2 constants with comments in the top block in `storage.go`. Wasn't entirely sure since I saw multiple const blocks across the file, e.g. [L619 ](https://github.com/googleapis/google-cloud-go/blob/main/storage/storage.go#L619-L622) and [L233](https://github.com/googleapis/google-cloud-go/blob/main/storage/storage.go#L619-L622)

Reply:
Ah well @tritone might have opinions but I usually keep constants in one spot per-file.